ipatests/prci_definitions/gating.yaml

```diff
@@ -242,3 +242,15 @@ jobs:
         template: *ci-master-f28
         timeout: 3600
         topology: *master_1repl
+
+ fedora-28/test_ntp_replacement_TestNTPoptions:
```

Comment:
This line is not aligned, it's missing a space char at the beginning. You can run `make yamllint` to check the syntax of yaml files.

Reply:
Checking